### ISSUE TITLE
data() popup is giving NPE when called from table

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#2246]
I have a dashboard that has a column defined as `data(${StreamId},1)`  which I am using to inspect at some cooked data.

The data is Records, and the dash shows the correct value for ${StreamId} in the column link

When I click on the link I get a blank data screen with no data, pressing the View Source button gives a Stream page with ID 1:1:1

In the log I have:
```
Error fetching child stream types for id 1, part number 0 java.lang.NullPointerException:
Cannot invoke "stroom.data.store.api.Source.get(long), because source is null at 
stroom.data.store.impl.DataFetcher.lambda$getAvailableChildStreamTypes$0 (DataFetcher.java:169)
```

**Comment by gcdev373** [comment]
Tested in `v7.0-beta.116` using dashboards and data created by `SetupSampleData`.  This resulted in expected outcome rather than reported failure.

Code inspection reveals that this NPE will occur when the meta data relating to the selected stream is not found.

Guard added and error message improved.

**Comment by stroomdev10** [comment]
looking into this a bit further, the function has the id and PartNo parameters switched rounnd.

Help suggests it should be id,PartNo. which seems right to me, so the func needs updating

**Comment by gcdev373** [comment]
Fixed on `7.01 branch
